File: LEETCODE/1470. 重新排列数组.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int[] res = new int[2*n];
        int index = 0;
        for(int i=0;i<n;i++){
            res[index] = nums[i];
            res[index+1] = nums[i+n];
            index+=2;
        }
        return res;
    }
}

File: LEETCODE/kotlin/1470. 重新排列数组.kt
class Solution {
    fun shuffle(nums: IntArray, n: Int): IntArray {
        var res = IntArray(2*n)
        var index = 0
        for (i in 0..n-1) {
            res[index] = nums[i]
            res[index+1] = nums[i+n]
            index += 2
        }
        return res
    }
}